lib/detect-testing-library-utils.ts

```diff
@@ -88,6 +90,15 @@ export function detectTestingLibraryUtils<
       context.settings['testing-library/filename-pattern'] ??
       DEFAULT_FILENAME_PATTERN;
 
+    /**
+     * Determines whether aggressive reporting is enabled or not.
+     *
+     * Aggressive reporting is considered as enabled when:
+     * - custom module is not set (so we need to assume everything
+     *    matching TL utils is related to TL no matter where it was imported from)
+     */
+    const isAggressiveReportingEnabled = () => !customModule;
```

Comment:
I extracted this into this simple function for adding some syntactic sugar around "aggressive reporting" concept. I'll create a separate PR soon where I'll use this in `isNodeComingFromTestingLibrary`.